static/template.palette

```diff
@@ -809,7 +809,7 @@
             "outputPorts": [],
             "key": -19,
             "fields": [],
-            "categoryType": "Other",
+            "categoryType": "Group",
```

Comment:
Good spot! I recall this was one of the first changes I made for service drops and added Service type to CategoryType but never used it. I'll test using "Service" and "Other" here and report back if there are quirks to either

Reply:
I can't seem to find a use of categoryType in daliuge, only in EAGLE. There is some logic around node links that checks for "Application" but otherwise both "Service" and "Other" both work